ocaml/default.nix

```diff
@@ -253,7 +254,7 @@ with oself;
     '';
   });
 
-  ctypes-0_17 = osuper.ctypes.overrideAttrs (_: {
+  ctypes-0_17 = osuper.ctypes.overrideAttrs (o: {
```

Comment:
if you make this inherit from `oself.ctypes` then you can inherit the fix from the other one